crates/biome_js_analyze/src/lint/complexity/no_useless_fragments.rs

```diff
@@ -66,7 +66,11 @@ declare_lint_rule! {
         version: "1.0.0",
         name: "noUselessFragments",
         language: "jsx",
-        sources: &[RuleSource::EslintReact("jsx-no-useless-fragment").same()],
+        sources: &[
+            RuleSource::EslintReact("jsx-no-useless-fragment").same(),
+            RuleSource::EslintReactX("no-useless-fragment").same(),
+            RuleSource::EslintReactXyz("no-useless-fragment").same()
+        ],
```

Comment:
This is more a provocation than a real question, but why are there three plugins that do the same thing? 😅